### ISSUE TITLE
No argument types in syntax

### ISSUE THREAD

**Issue by shaedrich** [foundeo/cfdocs#522]
In [arrayEach](https://cfdocs.org/arrayeach) function in syntax there are not only arguments in that function call, they are prefixed with their data types. That's inconsistent in different docs over cfdocs.

```coldFusion
arrayEach(array array, function(item, [index, [array]]) closure, [boolean parallel], [number maxThreads]) → returns void
```

should be

```coldFusion
arrayEach(array, function(item, [index, [array]]), [parallel], [maxThreads]) → returns void
```

right?

**Comment by pfreitag** [comment]
Right, the datatypes should be removed for consistency.

**Comment by shaedrich** [comment]
Okay. Seems like I have to do a cleanup.

**Comment by pfreitag** [comment]
Fixed in #526 thanks @shaedrich

**Comment by shaedrich** [comment]
You are welcome @pfreitag
